script.module.libmediathek4/lib/libmediathek4utils.py

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
+import xbmc
+import xbmcplugin
+import xbmcgui
+import xbmcaddon
+import xbmcvfs
+import sys
+
+temp = xbmc.translatePath(xbmcaddon.Addon().getAddonInfo('profile')+'temp')
+dict = xbmc.translatePath(xbmcaddon.Addon().getAddonInfo('profile')+'dict.py')
+
+
+
+def log(msg):
+	xbmc.log(msg)
+
+def getTranslation(id):
+	return xbmcaddon.Addon().getLocalizedString(id)
+			
+def pathUserdata(path):
+	special = xbmcaddon.Addon().getAddonInfo('profile')+path
+	special = special.replace('//','/').replace('special:/','special://')

```

Comment:
use xbmcvfs.validatePath(path) after https://github.com/xbmc/xbmc/pull/17750 goes in